tests/spread/integration/bind9-dnsutils/task.yaml

```diff
@@ -0,0 +1,12 @@
+summary: Integration tests for bind9-dnsutils
+
+execute: |
+  # Chisel a minimum number of slices to give us a runnable system that we can
+  # test in.
```

Comment:
Can we remove these comments since Canonical indicated in other PRs submitted they aren't needed here?